src/components/entity/update.vue

```diff
@@ -169,9 +169,7 @@ const currentVersion = computed(() =>
     overflow-y: auto;
   }
 
-  .modal-body { padding: 0; }
   table { margin-bottom: 0; }
-  .modal-actions { margin: 0; }
```

Comment:
Not super important because it's not user-facing, but `EntityBranchData` zeroes things out in a similar way to `EntityUpdate`.

Reply:
It's looking like this now:

<img width="923" alt="Screenshot 2025-04-23 at 4 22 49 PM" src="https://github.com/user-attachments/assets/948a1a83-be6c-41dd-a6e4-a7b4b602ef13" />